compiler/parser/Makefile

```diff
@@ -41,10 +45,26 @@ CLEAN_TARGS += \
 endif
 
 parser: FORCE
+	@rm -f bison-chapel.output bison-chapel.cpp flex-chapel.cpp
+	@rm -f ../include/bison-chapel.h ../include/flex-chapel.h
 	bison chapel.ypp
 	@if [ `grep "conflicts:" bison-chapel.output | wc -l` -ne 0 ]; then echo "PROBLEM: chapel.ypp contains conflicts"; exit 1; fi;
 	flex chapel.lex
 
+parser-update: parser-update-prep parser parser-update-compare
+
+parser-update-prep:
+	@[ -r bison-chapel.cpp ] && mv bison-chapel.cpp{,.tmp} || true
+	@[ -r flex-chapel.cpp  ] && mv flex-chapel.cpp{,.tmp} || true
+	@[ -r ../include/bison-chapel.h ] && mv ../include/bison-chapel.h{,.tmp} || true
+	@[ -r ../include/flex-chapel.h ] && mv ../include/flex-chapel.h{,.tmp} || true
+
+parser-update-compare:
+	@cmp -s bison-chapel.cpp{,.tmp} && mv bison-chapel.cpp{.tmp,} || rm -f bison-chapel.cpp.tmp
```

Comment:
The effect I am after is the hopefully earlier time stamp. Yes I can do it without the move, it would take me two commands (touch + remove) instead of one.